pkgs/tools/X11/xdg-utils/default.nix

```diff
@@ -45,7 +45,7 @@ stdenv.mkDerivation rec {
     substituteInPlace $out/bin/xdg-mime \
       --replace "/usr/bin/file" "${file}/bin/file"
 
-    sed 's# which # type -P #g' -i "$out"/bin/*
```

Comment:
This adds a space in front of things like:
```
`which "$command"` -> ` type -P "$command"`
```

Something like this covers all of the cases:
```diff
diff --git a/pkgs/tools/X11/xdg-utils/default.nix b/pkgs/tools/X11/xdg-utils/default.nix
index e7233d0..63073ad 100644
--- a/pkgs/tools/X11/xdg-utils/default.nix
+++ b/pkgs/tools/X11/xdg-utils/default.nix
@@ -46,6 +46,7 @@ stdenv.mkDerivation rec {
       --replace "/usr/bin/file" "${file}/bin/file"
 
     sed 's# which # type -P #g' -i "$out"/bin/*
+    sed 's#`which #`type -P #g' -i "$out"/bin/*
   '';
 
   meta = with stdenv.lib; {

```